test/napi/src/js/symbols.rs

```diff
@@ -0,0 +1,22 @@
+use neon::prelude::*;
+
+pub fn return_js_symbol_with_description(mut cx: FunctionContext) -> JsResult<JsSymbol> {
+    Ok(cx.symbol("neon:description"))
+}
+
+pub fn return_js_symbol(mut cx: FunctionContext) -> JsResult<JsSymbol> {
```

Comment:
Can you add a test for `JsSymbol::with_description` as well? Probably taking a `JsString` as an argument.